pkg/sectigo/mock/mock.go

```diff
@@ -30,31 +31,84 @@ import (
 
 var mockServer *Server
 
+type Profile struct {
+	ID      string
+	Name    string
+	Balance int
+	Params  []string
+}
+
+var subjectParams = [4]string{
+	"organizationName",
+	"localityName",
+	"stateOrProvinceName",
+	"countryName",
+}
+
+var nameParams = [3]string{
+	"commonName",
+	"dNSName",
+	"pkcs12Password",
+}
+
+// All the profiles that the mock knows about.
+var validProfiles = []Profile{
+	{
+		ID:      sectigo.ProfileIDCipherTraceEE,
+		Name:    sectigo.ProfileCipherTraceEE,
+		Balance: 50,
+		Params:  nameParams[:],
```

Comment:
Thanks for using the fixed array and converting it to a slice here; I think that's the better style for array "constants".